docs/rules/no-deprecated-functions.md

```diff
@@ -6,6 +6,10 @@ either been renamed for clarity, or replaced with more powerful APIs.
 While typically these deprecated functions are kept in the codebase for a number
 of majors, eventually they are removed completely.
 
+This rule requires knowing which version of Jest you're using - see
+[this section of the readme](README.md) for details on how that is obtained
```

Comment:
This link doesn't work, does it?

Reply:
No it doesn't - IntelliJ seems to have weird handling of relative links in nested docs :/